docs/topics/templates/document-attributes.adoc

```diff
@@ -30,6 +30,10 @@
 :MavenName: Maven plugin
 :MavenNameTitle: Maven Plugin
 
+:VscodePluginName: Visual Studio Code
+:VscodeNamePluginTitle: Visual Studio Code extension
+:VscodePluginShortName: VS Code
```

Comment:
It's the short name for Visual Studio Code, not the VS code plugin. I am also not sure about the wisdom of using an attribute whose name requires a lot more typing than the actual term.

We use attributes for words that could be replaced at a later date. I see no point in creating attributes for VS Code. It's not like someone is going to reuse a VS Code guide master.adoc for something else.